Source/WebKit/UIProcess/WebPageProxy.cpp

```diff
@@ -15194,6 +15194,16 @@ std::optional<IPC::AsyncReplyID> WebPageProxy::sendWithAsyncReplyToProcessContai
     );
 }
 
+template<typename M>
+void WebPageProxy::sendToProcessContainingFrameWithoutDestinationIdentifier(std::optional<FrameIdentifier> frameID, M&& message, OptionSet<IPC::SendOption> options)
+{
+    sendToWebPage(frameID,
+        [&message, options] (auto& targetPage) {
```

Comment:
Style guide nit; no space between capture list and arg list: https://webkit.org/code-style-guidelines/#spacing-lambda-paren